SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Security policy
+
+## Supported versions
+
+The available compatible releases are listed in [powsybl-dependencies](https://github.com/powsybl/powsybl-dependencies). In case of vulnerability fix, the main branches are first fixed. The fix will be available for next release train. In case of vulnerability of severity high/critical, the previous and more recent version is patched.
```

Comment:
```suggestion
The available compatible releases are listed in [powsybl-dependencies](https://github.com/powsybl/powsybl-dependencies). In case of vulnerability fix, the main branches are first fixed. The fix will be available for the next release train. In case of vulnerability of high or critical severity, the previous and more recent version is patched.
```